frontend/src/utils/StudentUtils.ts

```diff
@@ -7,14 +7,17 @@ import type {
 } from "../APIClients/types/TestClientTypes";
 import type { Answers } from "../types/AnswerTypes";
 import QuestionNumberTypes from "../types/QuestionNumberTypes";
+import type { Question, QuestionElement } from "../types/QuestionTypes";
```

Comment:
ig ideally in the future we refactor all our teacher data viz to use `QuestionElement` rather than `QuestionComponentResponse` lol

Reply:
eh. I think it would make sense, but not essential. I feel like we could maybe only refactor if there's some need for it? We probably have enough on our plate as is 😅

Reply:
yea fs, just a thought